synapse/lib/stormlib/compression.py

```diff
@@ -55,6 +56,7 @@ async def en(self, valu):
             mesg = f'Error during bzip2 compression - {str(e)}: {repr(valu)[:256]}'
             raise s_exc.StormRuntimeError(mesg=mesg) from None
 
+    @s_stormtypes.stormfunc(readonly=True)
```

Comment:
Could this potentially DOS the system if a malicious valu is provided?

Reply:
We can remove these markings for now; but the problem of a DOS with a malicious value exists regardless of readonly status and should probably be discussed separately.